README.md

```diff
@@ -35,51 +35,60 @@ opam pin add coq 8.10.2
 opam pin add coq-serapi 8.10.0+0.7.1
 ```
 
-### Installation of Python and PyTorch
+### Installation of PyTorch and Python libraries
 
-We recommend installing the required versions of Python and PyTorch
+We strongly recommend installing the required versions of Python and PyTorch
 using [Conda](https://docs.conda.io/en/latest/miniconda.html).
 
-Then, install PyTorch by following the instructions
-[here](https://pytorch.org/get-started/previous-versions/#v110), using
-the installation command suitable for your operating system, Python
-package manager, and whether you want to use it on a CPU or GPU.
-
-Next, either download a Roosterize [release][releases-link]
-or clone the Roosterize repository and enter the directory:
+First, create a Python 3.7 environment using Conda:
 ```
-git clone https://github.com/EngineeringSoftware/roosterize.git
-cd roosterize
+conda create --name roosterize python=3.7 pip
+conda activate roosterize
 ```
 
-To install other required Python libraries (`pip` is included with
-Python installation):
+Then, install PyTorch 1.1.0 by following the
+[official instructions](https://pytorch.org/get-started/previous-versions/#conda-5),
+using the command suitable for your operating system and whether you want to use
+it on a CPU or GPU. For example, the following command installs PyTorch for
+CPU-only use on Linux or Windows:
+```
+conda install pytorch-cpu==1.1.0 torchvision-cpu==0.3.0 cpuonly -c pytorch
+```
+Finally, install the required Python libraries:
 ```
-pip install -r requirements.txt
+conda install configargparse nltk future seutil==0.4.12 six torchtext==0.4.0 tqdm==4.30.*
 ```
```

Comment:
There is a problem here actually: seutil is not available on conda.

Alternatively, we can use pip instead of conda for everything. I can create requirements.{cpu,cuda9,cuda10}.txt files for them and they can be installed via `pip install -f`. How does that sound?

Reply:
This `conda` command is copied directly from what worked for me. In particular, when I type `conda list`, I see:
```
seutil                    0.4.12                   pypi_0    pypi
```
... so I'm pretty sure seutil is in conda 4.9.2.

The big issue with focusing on `pip` is that Ubuntu and other Linux distributions (and presumably macOS) *do not* give Python 3.7 by default anymore, but rather Python 3.8, meaning PyTorch will not be installable. Also, Ubuntu sometimes uses `pip` and sometimes `pip3`, adding to the confusion.

Reply:
These commands didn't work for me even if I upgrade to conda 4.9.2. According to my experience, that "pypi" likely means you've previously installed those packages via `pip install`. So I think we have to go with a mix of conda&pip approach...

Another suggestion to make it simpler for users. I can use `conda env export --from-history > env.yml` to save a conda environment specification, and users can simply use `conda env create -n NAME -f env.yaml`. How does that sound?

Reply:
Yes, a conda environment specification sounds like a big improvement.